ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -163,7 +163,7 @@ define([
                 if (this.get('currentUrl') === undefined) {
                     return false;
                 }
-               return /.*(localhost|org.codice.ddf.system.hostname).*/.test(this.getCurrentUrl().toString());
+               return /.*(localhost|org.codice.ddf.external.hostname).*/.test(this.getCurrentUrl().toString());
```

Comment:
❓ Is this check correct? Seems like we're looking for internal urls

Reply:
Good catch, fixing this